@ndlib/gatsby-plugin-marble-elasticsearch/gatsby-node.js

```diff
@@ -29,17 +27,15 @@ exports.onPostBuild = async (
   }
 
   const options = {
-    host: url,
-    port:443,
-    protocol:'https',
-    connectionClass: auth,
-    requestTimeout: '2000000000',
-    awsConfig: new AWS.Config({ region: region }),
+    node: url,
+    ssl: {
+      rejectUnauthorized: false,
```

Comment:
If you are always authenticating, and using https, do you need to set rejectUnauthorized to false?